.github/workflows/ci.yml

```
@@ -0,0 +1,13 @@
+on: push
```

```suggestion
on: [push, pull_request]
```
I'm not 100% sure about this but maybe needed for PRs from forks?

---

No clue, but I don't see how it could hurt. I'd be only careful about jobs starting twice, once for the `push` trigger and once for the `pull_request` trigger. I guess we'll see :smile:

---

I checked for example here: https://github.com/nvim-telescope/telescope-file-browser.nvim/blob/master/.github/workflows/lint.yml
where the same is done, but yeah lets see :)

I'm tbh more familiar with gitlab ci :)